instant-xml-macros/src/lib.rs

```diff
@@ -27,228 +24,6 @@ pub fn from_xml(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
     de::from_xml(&ast).into()
 }
 
-pub(crate) use container::{ContainerMeta, Mode};
```

Comment:
Not a fan of this change. IMO 100-200 line files are a net negative to my ability to understand the code, my optimal size is around 300-800 lines of non-test code.

Reply:
Understand. For me it a visibility modefier reason to limit types and their impl blocks to their own module, with quite a few exceptions of course. Easier to reason about them then. Save thing as with star imports, ok if i flatten things in the end?

Reply:
Similarly to the other aspect, I don't want this right now. If you end up adding substantial logic for one of these types we can definitely consider splitting it out at that point.